src/wp-content/themes/twentynineteen/inc/color-patterns.php

```diff
@@ -48,24 +48,24 @@ function twentynineteen_custom_colors_css() {
 	$lightness = absint( $lightness ) . '%';
 
 	/**
-	 * Filters Twenty Nineteen default hover lightness level.
+	 * Filters Twenty Nineteen default reduced lightness level.
 	 *
 	 * @since Twenty Nineteen 1.0
 	 *
-	 * @param int $lightness_hover Hover color lightness level.
+	 * @param int $lightness_reduced Reduced color lightness level.
 	 */
-	$lightness_hover = apply_filters( 'twentynineteen_custom_colors_lightness_hover', 23 );
-	$lightness_hover = absint( $lightness_hover ) . '%';
+	$lightness_reduced = apply_filters( 'twentynineteen_custom_colors_lightness_reduced', 23 );
+	$lightness_reduced = absint( $lightness_reduced ) . '%';
 
 	/**
-	 * Filters Twenty Nineteen default selection lightness level.
+	 * Filters Twenty Nineteen default increased lightness level.
 	 *
 	 * @since Twenty Nineteen 1.0
 	 *
-	 * @param int $lightness_selection Selection color lightness level.
+	 * @param int $lightness_increased Increased color lightness level.
 	 */
-	$lightness_selection = apply_filters( 'twentynineteen_custom_colors_lightness_selection', 90 );
-	$lightness_selection = absint( $lightness_selection ) . '%';
+	$lightness_increased = apply_filters( 'twentynineteen_custom_colors_lightness_increased', 90 );
+	$lightness_increased = absint( $lightness_increased ) . '%';
```

Comment:
Renamed `$lightness_hover` to be more generic as it's used for hover, active and focus, and now secondary border color too.

Renamed `$lightness_selection` for consistency.

Reply:
If these filters are renamed they will break for people who are using them.

Reply:
Reverted